x/crosschain/keeper/msg_server_add_to_outtx_tracker.go

```diff
@@ -48,7 +48,7 @@ func (k msgServer) AddToOutTxTracker(goCtx context.Context, msg *types.MsgAddToO
 		return &types.MsgAddToOutTxTrackerResponse{IsRemoved: true}, nil
 	}
 
-	isEmergencyGroup := k.GetAuthorityKeeper().IsAuthorized(ctx, msg.Creator, authoritytypes.PolicyType_groupEmergency)
+	isEmergencyGroup, _ := k.GetAuthorityKeeper().IsAuthorized(ctx, msg)
```

Comment:
Why skipping errors in these?

Reply:
We should not return on error here as the message can be signed by an observer also .